lib/qesapdeployment.pm

```diff
@@ -745,4 +749,135 @@ sub qesap_get_az_resource_group {
     return $result;
 }
 
+=head3 qesap_get_az_address_range
+
+Query the existing maintenance deployments for azure
+and determine a name with a free adress range number
+for the current deployment
+
+=cut
+
+sub qesap_get_az_address_range {
+    my $worker = get_var("WORKER_INSTANCE");
```

Comment:
I suggest to move `get_var("WORKER_INSTANCE")` in the caller. Add an integer argument here named `slot`